ovh/resource_vrack_dedicated_cloud_datacenter_test.go

```diff
@@ -0,0 +1,86 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/ovh/go-ovh/ovh"
+)
+
+func init() {
+	resource.AddTestSweepers("ovh_vrack_dedicated_cloud_datacenter", &resource.Sweeper{
+		Name: "ovh_vrack_dedicated_cloud_datacenter",
+		F:    testSweepVrackDedicatedCloudDatacenter,
+	})
+}
+
+func testSweepVrackDedicatedCloudDatacenter(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	serviceName := os.Getenv("OVH_VRACK_SERVICE_TEST")
+	if serviceName == "" {
+		log.Print("[DEBUG] OVH_VRACK_SERVICE_TEST is not set. No vrack_dedicated_cloud_datacenter to sweep")
+		return nil
+	}
+
+	dedicatedCloudDatacenter := os.Getenv("OVH_DEDICATED_CLOUD_DATACENTER_TEST")
+	if dedicatedCloudDatacenter == "" {
+		log.Print("[DEBUG] OVH_DEDICATED_CLOUD_DATACENTER_TEST is not set. No vrack_dedicated_cloud_datacenter to sweep")
+		return nil
+	}
+
+	endpoint := fmt.Sprintf("/vrack/%s/dedicatedCloudDatacenter/%s",
+		url.PathEscape(serviceName),
+		url.PathEscape(dedicatedCloudDatacenter),
+	)
+
+	var result interface{}
+	if err := client.Get(endpoint, &result); err != nil {
+		if errOvh, ok := err.(*ovh.APIError); ok && errOvh.Code == 404 {
+			return nil
+		}
+		return err
+	}
+
+	task := VrackTask{}
+	if err := client.Delete(endpoint, &task); err != nil {
+		return fmt.Errorf("Error calling DELETE %s with %s/%s:\n\t %q", endpoint, serviceName, dedicatedCloudDatacenter, err)
+	}
+	if err := waitForVrackTask(&task, client); err != nil {
+		return fmt.Errorf("Error waiting for vrack (%s) to detach dedicatedCloudDatacenter (%s): %s", serviceName, dedicatedCloudDatacenter, err)
+	}
+
+	return nil
+}
+
+func TestAccVrackDedicatedCloudDatacenter_basic(t *testing.T) {
```

Comment:
not sure this test works ? I think `ImportState` and other import-related configs are missing from your test step since resource cannot be created

Reply:
just this comment remaining, otherwise the PR is ok